src/sgame/sg_bot_nav.cpp

```diff
@@ -590,13 +590,13 @@ static void BotFindDefaultSteerTarget( gentity_t *self, glm::vec3 &dir )
 
 	// We add a delta determined from clientNum to avoid bots
 	// dancing in sync
-	int time = (110 * self->num() + self->client->time1000) % 1000;
+	int time = (110 * self->num() + self->client->time10000) % 1600;
 
-	// 280 instead of 700/2=350 because we want some left/right
+	// 600 instead of 800 because we want some left/right
```

Comment:
Comment doesn't match the code which does 800 and symmetrical

Reply:
it's 600 on the line below, but I've just moved the comment around for clarity